classes/patreon_frontend.php

```diff
@@ -305,14 +292,49 @@ public static function patreonMakeUniversalButton($min_cents=false,$state=false,
 		
 		$state['final_redirect_uri'] = $final_redirect;
 		
-		$href = self::MakeUniversalFlowLink($send_pledge_level,$state,$client_id);
+		// $href = self::MakeUniversalFlowLink($send_pledge_level,$state,$client_id);
```

Comment:
same goes here.

Reply:
Same as earlier:

In a near opportunity we'll clean up various variables initialized in those functions which were used for MakeUniversalFlowLink only. They are left there as a comment for that purpose.  Lets keep them 1-2 weeks until that cleanup happens.